test/__snapshots__/OracleLibrary.spec.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`OracleLibrary #consult gas test 1`] = `6129`;
```

Comment:
that's ok this isnt realistic. we know what observe costs from core tests so this tells us the overhead of the code in the library